src/lib/validators.ts

```diff
@@ -1,28 +1,31 @@
-'use strict';
 import * as Bluebird from 'bluebird';
 import { stat } from 'fs';
 import { isFQDN, isPort } from 'validator';
 
 const statAsync = Bluebird.promisify(stat);
 
+/**
+ * Validate a directory exists
+ * @param {string} path
+ * @param {boolean} raise
+ * @returns {Promise<boolean>}
+ */
 export async function validateDirectoryPath(
 	path: string,
 	raise = true,
 ): Promise<boolean> {
 	try {
 		const stat = await statAsync(path);
```

Comment:
Don't use `sync`d function, `await` the asyncd.

Reply:
I was inclined to use await instead of sync calls, as per your previous review comments:
[here](https://github.com/balena-io/katapult/pull/25#discussion_r284267485) and [here](https://github.com/balena-io/katapult/pull/25#discussion_r284277941). Should we change that?

Reply:
Validators are dropped, as we are now using schema everywhere.